src/client/datascience/notebookStorage/baseModel.ts

```diff
@@ -163,16 +164,28 @@ export function getDefaultNotebookContent(pythonNumber: number = 3): Partial<nbf
  * If a preferred language is provided we use that.
  * We do not default to Python, as selecting a kernel will update the language_info in the ipynb file (after a kernel is successfully started).
  */
-export function getDefaultNotebookContentForNativeNotebooks(language?: string): Partial<nbformat.INotebookContent> {
-    const metadata: undefined | nbformat.INotebookMetadata = language
-        ? {
-              language_info: {
-                  name: language,
-                  nbconvert_exporter: 'python'
-              },
-              orig_nbformat: 2
-          }
-        : undefined;
+export function getDefaultNotebookContentForNativeNotebooks(language: string = ''): Partial<nbformat.INotebookContent> {
+    let metadata: undefined | nbformat.INotebookMetadata;
+    switch (language.toLowerCase()) {
+        case '':
+            break;
+        case PYTHON_LANGUAGE.toLowerCase():
+            metadata = {
+                language_info: {
+                    name: language,
+                    nbconvert_exporter: 'python'
+                },
+                orig_nbformat: 2
+            };
+            break;
+        default:
+            metadata = {
```

Comment:
Fixed bug, when creating a C# notebook (we shouldn't be adding `nbconvert_exporter`)